docs: Add info about Metadata Registry (#110)
oasisprotocol/docs: docs/general/run-a-node/set-up-your-node/run-validator.md

:::

## Oasis Metadata Registry

> Congratulations, if you made it this far, you've properly connected your node to the network and became a validator on the Oasis Network.

Reviewer: Wrap the line to 80 characters at least in this section.

For the final touch, you can add some metadata about your entity to the [Metadata Registry]. The Metadata Registry is the same for the Mainnet and the Testnet. 

> 

Reviewer: I would join the paragraphs.

```suggestion
```

:::

## Oasis Metadata Registry

> Congratulations, if you made it this far, you've properly connected your node 

Reviewer: This paragraph appears out of place now. Can you move it before the Oasis Metadata Registry title?

:::

## Oasis Metadata Registry

> Congratulations, if you've made it this far, you've properly connected your node to the network and you're now a validator on the Oasis Network.

Reviewer: ```suggestion
Congratulations, if you made it this far, you've properly connected your node to the network and became a validator on the Oasis Network.
```

> For the final touch, you can add metadata about your entity to the [Metadata Registry](https://github.com/oasisprotocol/metadata-registry). The Metadata Registry is the same for the Mainnet and the Testnet. Metadata consists of entity name, email, Keybase handle, Twitter handle, etc. This information is used also by third party applications. For example [Oasis Scan](https://www.oasisscan.com/validators) fetches image from keybase and uses it as a node operator's avatar.

Reviewer: Please trim the line length to 80 characters or so. Replace links with anchors and define them below the paragraph:

```suggestion
For the final touch, you can add some metadata about your entity to the [Metadata Registry]. The Metadata Registry is the same for the Mainnet and the Testnet. The metadata consists of your entity name, email, Keybase handle, Twitter handle, etc. This information is also used by third party applications. For example [Oasis Scan] fetches your image from Keybase and uses it as the node operator's avatar.

[Metadata Registry]: https://github.com/oasisprotocol/metadata-registry
[Oasis Scan]: https://www.oasisscan.com/validators
```
